project_scrum/project_scrum.py

```diff
@@ -0,0 +1,500 @@
+    color = fields.Integer(related='project_id.color')
+    description = fields.Html(string='Description')
+    description_short = fields.Text(compute='_conv_html2text', store=True)
+    actor_ids = fields.Many2many(comodel_name='project.scrum.actors', string='Actor')
+    project_id = fields.Many2one(comodel_name='project.project', string='Project', ondelete='set null',
+        select=True, track_visibility='onchange', change_default=True)
+    sprint_ids = fields.Many2many(comodel_name='project.scrum.sprint', string='Sprint', store=True)
+    task_ids = fields.One2many(comodel_name='project.task', inverse_name='us_id')
+    task_test_ids = fields.One2many(comodel_name='project.scrum.test', inverse_name='user_story_id_test')
+    task_count = fields.Integer(compute='_task_count', store=True)
+    test_ids = fields.One2many(comodel_name='project.scrum.test', inverse_name='user_story_id_test')
+    test_count = fields.Integer(compute='_test_count', store=True)
+    sequence = fields.Integer('Sequence')
+    company_id = fields.Many2one(related='project_id.analytic_account_id.company_id')
+    moscow = fields.Selection([
+            (1, 'must'), 
+            (2, 'should'), 
+            (3, 'could'), 
+            (4, 'wont'), 
+            ('not_set', 'Not Set'), 
+            ], string='Moscow')
+    value = fields.Selection([
+             (0, '0'),
+             (0.5, '0.5'),
+             (1, '1'),
+             (2, '2'),
+             (3, '3'), 
+             (5, '5'), 
+             (8, '8'), 
+             (13, '13'), 
+             (20, '20'),
+             (40, '40'),
+             (100, '100'),
+             (00, 'Not Set'),
+             ], string='Value')
+    risk = fields.Selection([
+            (0, '0'),
+            (1, '1'),
+            (2, '2'),
+            (3, '3'),
+            (5, '5'),
+            (8, '8'),
+            (13, '13'),
+            (20, '20'), 
+            (40, '40'),
+            (100, '100'),
+            (00, 'Not Set'),
+            ], string='Risk')
+    kano = fields.Selection([
+            ('excitement', 'Excitement'),
+            ('indifferent', 'Indifferent'),
+            ('mandatory', 'Mandatory'), 
+            ('performance', 'Performance'),
+            ('questionable', 'Questionable'),
+            ('reverse', 'Reverse'), 
+             ('not_set', 'Not Set'),
+            ], string='Kano')
+    reference = fields.Char('Number', select=True, readonly=True, copy=False,default='/')
+    kanban_state = fields.Selection([('normal', 'Mark as impeded'), ('blocked', 'Mark as waiting'), 
+                                    ('done', 'Mark item as defined and ready for implementation')], 'Kanban State', default='blocked')
+    
+    @api.one
+    def _conv_html2text(self):  # method that return a short text from description of user story
+        for d in self: 
+            d.description_short = re.sub('<.*>', ' ', d.description or '')
```

Comment:
use https://github.com/OCA/OCB/blob/10.0/odoo/tools/mail.py#L263 (if unknown, the whole odoo.tools namespace is useful to read)